sdk/search/azure-search-documents/README.md

```diff
@@ -2,11 +2,11 @@
 
 Azure Cognitive Search is a fully managed cloud search service that provides a rich search experience to custom applications.
 
-[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/search/azure-search) |
-[Package (PyPI)](https://pypi.org/project/azure-search/) |
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/search/azure-search-documents) |
```

Comment:
orthogonal but in general we try to avoid links to master. Instead they should be relative path links in the repo.  For reference see https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md.

Reply:
@weshaggard I have updated all the links to master *except* this one. As a user if a link is to GH source code I would never expect it to go anywhere except master

Reply:
relative links still work best here as it protects against renames/moves as well as deleting in the future. As part of our publishing process we replace relative links to absolute links based on the sha so it will always be stable overtime, were as using a branch like master might break in the future.

Reply:
Please note that the README.md is included in the (distribution) packages.

Reply:
Yes which is why we re-write these links relative links as pre-step before building the package so that they are replaced with stable fully qualified links based on the sha.